atom/browser/browser_mac.mm

```diff
@@ -12,6 +12,7 @@
 #include "base/mac/bundle_locations.h"
 #include "base/mac/foundation_util.h"
 #include "base/strings/sys_string_conversions.h"
+#include "base/strings/string_number_conversions.h"
```

Comment:
This should go above `base/strings/sys_string_conversions.h` for alphabetical sorting